.travis.yml

```diff
@@ -10,11 +10,12 @@ addons:
       # Needed for NetMHC
       - tcsh
 cache:
-  pip: true
-  # cache directory used for Ensembl downloads of GTF and FASTA files
+  # cache pip files, also directory used for Ensembl downloads of GTF and FASTA files
```

Comment:
I don't see anything like that in the current documentation at https://docs.travis-ci.com/user/caching/, figured we should be consistent